.github/workflows/windows.yml

```diff
@@ -97,6 +97,9 @@ jobs:
     - name: Install getopt (vcpkg)
       if: steps.cache-vcpkg.outputs.cache-hit != 'true'
       run: .\vcpkg\vcpkg install getopt:x64-windows
+    - name: Install curl (vcpkg)
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: .\vcpkg\vcpkg install curl:x64-windows
```

Comment:
woops, you need to run it once without checking for the cache, as curl is not cached, and them update the PR adding the verification.

Reply:
Solved in #409